frontends/p4/inlining.h

```diff
@@ -225,7 +225,9 @@ class GeneralInliner : public AbstractInliner<InlineList, InlineSummary> {
     }
     // controlled visiting order
     const IR::Node* preorder(IR::MethodCallStatement* statement) override;
-    template<class T> void inline_subst(T *caller, IR::IndexedVector<IR::Declaration> T::*locals);
+    template<class T, class U>
+    void inline_subst(T *caller, IR::IndexedVector<IR::Declaration> T::*locals,
```

Comment:
Please document the U argument.

Reply:
I've replaced `T` and `U` with `P4Block` and `P4BlockType`, matching the definition in the cpp file. I've also copied the comment from the cpp file into the header.